src/specialitensornetworks.jl

```diff
@@ -0,0 +1,92 @@
+#RETURN THE IDENTITY TENSOR NETWORK (I.E. NETWORK WITH IDENTITY TENSORS ON EACH SITE) FOR A GIVEN GRAPH
+function identityITensorNetwork(g::NamedDimGraph; link_space = 2)
```

Comment:
Maybe there is a better name for this, like `delta_network`? I wonder if this is alreadt called something in the literature. Maybe `copy_tensor_network`.

Reply:
Also maybe to avoid code duplication between this version and the one below, this could just make an `IndsNetwork` and call the one below (which I assume is supposed to accept an `IndsNetwork`?).

Reply:
Good idea, I have renamed to delta_network and avoided the code duplication.